R/postprocess.R

```diff
@@ -403,3 +378,74 @@ enw_quantiles_to_long <- function(posterior) {
   long[, quantile := as.numeric(quantile) / 100]
   return(long[])
 }
+
+#' Build the ord_obs `data.table`.
+#'
+#' @param obs Observations as pulled from nowcast$latest[[1]].
+#' @param max_delay Whole number representing the maximum delay
+#' in units of the timestep.
+#' @param timestep The timestep to be used. This can be a string
+#' ("day", "week", "month") or a numeric whole number representing
+#' the number of days.
+#' @param sample String, "get_sample" or "no_sample".
+#' @param nowcast If getting posterior samples, the fit to pull
+#' the draws from.
+#'
+#' @return ord_obs A `data.table`.
+#'
+#' @family postprocess
+
+build_ord_obs <- function(obs, max_delay, timestep, sample, nowcast = NULL) {
+    internal_timestep <- get_internal_timestep(timestep)
```

Comment:
as we still need this in the function that wraps this perhaps we should just pass in internal timestep?

Reply:
yeah you're right - felt wrong to pass in both timestep and internal timestep but it's cleaner